src/api/server/common.rst

```diff
@@ -1586,6 +1586,75 @@ node, you can use:
 This returns an entire statistics object, as with the full request, but
 containing only the requested individual statistic.
 
+============================
+``/_node/{node-name}/_prometheus``
+=============================
+
+.. http:get:: /_node/{node-name}/_prometheus
+    :synopsis: Returns server statistics in prometheus format
+
+    The ``_prometheus`` resource returns a text/plain response that consolidates our
+    `_stats`, `active_task`, and `_system` endpoints. The format is determined by
+    https://prometheus.io/docs/introduction/overview/. Currently the format version 2.0.
```

Comment:
I feel like this last sentence is missing a word?  "Currently the format version is 2.0" perhaps?

Also, we can probably remove the word "Currently", as I think that's implicit.